src/cf-base.less

```diff
@@ -140,70 +140,82 @@ body {
     line-height: @base-line-height;
 }
 
-.heading-1( @fs: 34px, @line-height: 44px ) {
+.heading-1( @fs: 34px ) {
     @font-size: @fs;
     .webfont-regular();
-    // Spacing in the design manual is based off of the baseline so this margin
-    // number has been eyeballed to be 30px.
-    margin-bottom: unit( 16px / @font-size, em );
+
+    margin-bottom: unit( 20px / @font-size, em );
     font-size: unit( @font-size / @base-font-size-px, em );
-    line-height: unit( @line-height / @font-size );
+    line-height: 1.25;
 }
 
-.heading-2( @fs: 26px, @line-height: 33px ) {
+.heading-2( @fs: 26px ) {
     @font-size: @fs;
     .webfont-regular();
-    // Spacing in the design manual is based off of the baseline so this margin
-    // number has been eyeballed to be 30px.
-    margin-bottom: unit( 19px / @font-size, em );
+
+    margin-bottom: unit( 20px / @font-size, em );
     font-size: unit( @font-size / @base-font-size-px, em );
-    line-height: unit( @line-height / @font-size );
+    line-height: 1.25;
+
+    p ~ & {
```

Comment:
For the combinations, in our demo there was a list before the heading, but it could have easily been an image, figure, or blockquote. For normal posts, WP does wrap these in paragraphs, but now that we're switching CMS and because content is often derived from templates, we can't guarantee it'll always be a paragraph prior to the heading.

For your example, my understanding from what we discussed with @nataliafitzgerald is that it really should be 45 in that situation and the designer shouldn't have spec'd 30px, but I agree if that's not the case, this will be an issue. 

Maybe we really need to include a wrapper like v1 is already doing w/ `.post_body`. That'll allow designers to set a custom space above for any situation outside of regular running text.